Formula/zero-install.rb

```diff
@@ -95,6 +102,9 @@ def install
     archives = opamroot/"repo/default/archives"
     modules = []
     resources.each do |r|
+      if build.without? "gtk+"
```

Comment:
I tried something _like this_ & Rubocop yelled at me. Perhaps I wrote it slightly wrong though, will try this way.